sportsdataverse/cfb/cfb_pbp_fox.py

```
@@ -0,0 +1,479 @@
+the sdv-internal-refs repo under ``_notes/foxsportsapi/cfb_pbp_backup/``.
+"""
+
+from __future__ import annotations
+
+import re
+from typing import Any, Dict, List, Optional
+
+from sportsdataverse.cfb.cfb_fox_ext import _fox_get
+from sportsdataverse.cfb.cfb_pbp import CFBPlayProcess
+
+__all__ = ["fox_to_espn_summary", "fox_cfb_play_process"]
```

Ruff (0.15.15)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)